hail/src/main/scala/is/hail/io/RowStore.scala

```diff
@@ -1665,15 +1665,15 @@ class RichContextRDDRegionValue(val crdd: ContextRDD[RVDContext, RegionValue]) e
     val nPartitions = crdd.getNumPartitions
     val d = digitsNeeded(nPartitions)
 
-    val fullRowType = t.rvRowType
-    val fullRowPType = fullRowType.physicalType
-    val rowsRVType = t.rowType
-    val localEntriesIndex = t.entriesIdx
-    val entriesRVType = t.entriesRVType
+    val fullRowType = t.rowType
```

Comment:
This looks wrong.  I thought "full" meant including entries, but t.rowType doesn't include entries.  Maybe you just need to rename fullRowType to rowType?

Reply:
t is the RVDType, so the rowType does include entries

Reply:
then I use the MatrixType operations to get out the entries-only and no-entries types